src/services/auth.service.ts

```diff
@@ -1,16 +1,43 @@
+import logger from '../logger/logger';
 import { AccessToken, APIResponse, PhoneNumber } from '../types';
 
 import { APIService } from './api-service';
 
 export class AuthService extends APIService {
   sendOTP = async (phoneNumber: PhoneNumber): Promise<APIResponse> => {
-    return this.post('/accounts', { phoneNumber });
+    try {
+      const payload = {
+        phone_number: {
+          country_code: phoneNumber.countryCode.startsWith('+')
+            ? phoneNumber.countryCode
+            : `+${phoneNumber.countryCode}`,
+          phone_number: phoneNumber.phoneNumber,
+        },
+      };
+
+      return this.post('/accounts', payload);
+    } catch (error) {
+      logger.error(`SendOTP service error: ${error}`);
+      throw error;
+    }
   };
 
   verifyOTP = async (otp: string, phoneNumber: PhoneNumber): Promise<APIResponse> => {
-    return this.post<AccessToken>('/access-tokens', {
-      otpCode: otp,
-      phoneNumber,
-    });
+    try {
+      const payload = {
+        phone_number: {
+          country_code: phoneNumber.countryCode.startsWith('+')
```

Comment:
In the UI, we have country code selector I believe which means we should always have country code. There should be no 2 way to do this.